bucket/simple-dnscrypt.json

```
@@ -0,0 +1,62 @@
+{
+    "version": "0.7.1",
+    "description": "An easy to use GUI for dnscrypt-proxy",
+    "homepage": "https://simplednscrypt.org/",
+    "license": "MIT",
+    "suggest": {
+        "vcredist2017": "extras/vcredist2017"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/bitbeans/SimpleDnsCrypt/releases/download/0.7.1/SimpleDNSCrypt64.msi",
+            "hash": "5BA0C94774ACD7ADE18446B4351E7D28A948C490AEFC50DC6C08C13551F19F7F",
+            "post_install": [
+                "if (Test-Path \"$dir\\SimpleDNSCrypt64.msi\") {Remove-Item \"$dir\\SimpleDNSCrypt64.msi\" | Out-Null}"
+            ]
+        },
+        "32bit": {
+            "url": "https://github.com/bitbeans/SimpleDnsCrypt/releases/download/0.7.1/SimpleDNSCrypt.msi",
+            "hash": "CAC6B8119AF3DA70A5977A2E73EE5214303990FEDD7ABBC5F247A1E2CF19A663",
+            "post_install": [
+                "if (Test-Path \"$dir\\SimpleDNSCrypt.msi\") {Remove-Item \"$dir\\SimpleDNSCrypt.msi\" | Out-Null}"
+            ]
+        }
+    },
+    "pre_install": [
+        "if (!(is_admin)) { error \"$app requires admin rights to $cmd\"; break }",
+        "",
+        "if (!(Test-Path \"$env:LOCALAPPDATA\\SimpleDnsCrypt\")) {New-Item -Path \"$env:LOCALAPPDATA\" -Name \"SimpleDnsCrypt\" -ItemType \"directory\" -Force | Out-Null}"
+    ],
+    "shortcuts": [
+        [
+            "SimpleDnsCrypt.exe",
+            "Simple DnsCrypt"
+        ]
+    ],
+    "bin": "SimpleDnsCrypt.exe",
+    "persist": [
+        "dnscrypt-proxy\\"
+    ],
```

```suggestion
    "persist": "dnscrypt-proxy\\dnscrypt-proxy.toml",
```
If we *persist* the whole folder, it will block the update of `dnscrypt-proxy` when user updates the app.

I tried tweaking the config, and found the only file that is related to user config is `dnscrypt-proxy.toml`. Other files such as `public-resolvers.md` is updated according to the data on the app's server.

---

Hey Isaac, I going to be heading to bed now. I will tell you when my response to the issue later.

---

> Hey Isaac, I going to be heading to bed now. I will tell you when my response to the issue later.

No worries. We are not in a rush.

---

I fixed it, check the latest commit.